docs/make.jl

```diff
@@ -52,14 +53,49 @@ for (i, f) in enumerate(pluto_files)
     global TutorialMenu
     @info "Building Pluto Notebook $f.jl"
     pluto_file = pluto_src_folder * f * ".jl"
-    s = Pluto.ServerSession()
-    nb = Pluto.SessionActions.open(s, pluto_file; run_async=false)
-    write(pluto_output_folder * f * "_pluto.html", Pluto.generate_html(nb))
+    html = notebook2html(pluto_file)
     write(
         pluto_output_folder * f * ".md",
         """
+        ```@meta
+        EditURL = "$(pluto_src_folder)$(f).jl"
+        ```
         ```@raw html
-        <iframe style="border:none; width:100%; height: $(pluto_heights[i])rem;" src="$(f)_pluto.html"></iframe>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/fonsp/Pluto.jl@0.16.4/frontend/treeview.css" type="text/css" />
+        <style>
+        div.markdown {
+            padding-top: 1rem;
+            padding-bottom: 2rem;
+        }
+        </style>
+        $html
+        <script>
+  </script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css" integrity="sha384-R4558gYOUz8mP9YWpZJjofhk+zx0AS11p36HnD2ZKj/6JR5z27gSSULCNHIRReVs" crossorigin="anonymous">
+
+    <!-- The loading of KaTeX is deferred to speed up page rendering -->
+    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js" integrity="sha384-z1fJDqw8ZApjGO3/unPWUPsIymfsJmyrDVWC8Tv/a1HeOtGmkwNd/7xUS0Xcnvsx" crossorigin="anonymous"></script>
+
+    <!-- To automatically render math in text elements, include the auto-render extension: -->
+    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"
+        onload="renderMathInElement(document.body);"></script>
+        
+  <script>
+  const options = {
+    delimiters: [
+      {left: "$$", right: "$$", display: true},
+      {left: "$", right: "$", display: false},
```

Comment:
```suggestion
      {left: "\$\$", right: "\$\$", display: true},
      {left: "\$", right: "\$", display: false},
```
Something like this might solve the issue, but I haven't tested it.